### PR TITLE
[REALM-4] Refactored falcon implementation to PEP-8 standards, cleaned up code

### PR DESCRIPTION
**Pull request by daniellund14**

### CODE REVIEW

File: onthequad/db_config.py [hunk 1]
```diff
@@ -14,19 +17,23 @@ def config():
...
+    firebase = DataBaseSetUp().config()
+    authen = firebase.auth()
+    return authen
 
-    def authentication():
 
-        firebase = DataBaseSetUp.config()
-        authen = firebase.auth()
-        return authen
+# def noquote(self):
```
Comment (paugajt) [inline onthequad/db_config.py]: Do we need this here? Or can we just delete the commented out code?

File: onthequad/user_resource.py [hunk 2]
```diff
@@ -28,44 +31,46 @@ def on_post(self, req, resp):
...
-                        aUser : userObject.__dict__
+                        a_user : user_object.__dict__
                     }
-                    resp.body = json.dumps(sendBack)
+                    resp.body = json.dumps(send_back)
                     resp.status = falcon.HTTP_200
 
-                except:
+                # TODO figure out what this exception is and catch it instead of general Exception
+                except Exception:
```
Comment (paugajt) [inline onthequad/user_resource.py]: this exception is used when someone is trying to create an account with an email that is already associated with a users account

File: onthequad/user_resource.py [hunk 3]
```diff
@@ -28,44 +31,46 @@ def on_post(self, req, resp):
...
                     send = {"message": "EMAIL_EXISTS"}
                     resp.body = json.dumps(send)
                     resp.status = falcon.HTTP_400
                     return
-
-            except:
+            # TODO figure out what this exception is and catch it instead of general Exception
```
Comment (paugajt) [inline onthequad/user_resource.py]: This exception is very generic but is thrown when there are missing fields in the json

File: onthequad/users_db.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+from onthequad import db_config
+
+
+class UsersDB(object):
```
Comment (paugajt) [inline onthequad/users_db.py]: Should this be `users_db`?

File: onthequad/util/profanity_filter.py [hunk 5]
```diff
@@ -0,0 +1,356 @@
+
+
+def profanity_filter(payload_dict):
+	for key, value in payload_dict.items():
+
+		if value in arrBad:
```
Comment (paugajt) [inline onthequad/util/profanity_filter.py]: Should we rename `arrBad` to something more descriptive?

File: onthequad/util/profanity_filter.py [hunk 6]
```diff
@@ -0,0 +1,356 @@
...
+			print (key + " has a bad word in it.")
```
Comment (paugajt) [inline onthequad/util/profanity_filter.py]: And remove print statements

File: onthequad/util/user_helper.py [hunk 7]
```diff
@@ -1,6 +1,7 @@
 
 class UserHelper:
```
Comment (paugajt) [inline onthequad/util/user_helper.py]: The class name should be renamed as well?

File: onthequad/db_config.py (hunk 1 above)
Comment (daniellund14) [reply onthequad/db_config.py]: I will remove, it wasn't commented out before and looks to be unused

File: onthequad/user_resource.py (hunk 2 above)
Comment (daniellund14) [reply onthequad/user_resource.py]: So I mean the underlying exception so we are just not catching a general exception, that can be a little too vague. Not a high priority but can remove the TODO if that is the only exception being thrown

File: onthequad/user_resource.py (hunk 3 above)
Comment (daniellund14) [reply onthequad/user_resource.py]: Same here, thinking this is very generic. If it is the case that it is the only exception thrown I will add a comment

File: onthequad/users_db.py (hunk 4 above)
Comment (daniellund14) [reply onthequad/users_db.py]: PEP-8 standard makes Classes still have the CapWord case https://www.python.org/dev/peps/pep-0008/#naming-conventions

File: onthequad/util/profanity_filter.py (hunk 5 above)
Comment (daniellund14) [reply onthequad/util/profanity_filter.py]: Will do

File: onthequad/util/profanity_filter.py (hunk 6 above)
Comment (daniellund14) [reply onthequad/util/profanity_filter.py]: Yes, would like to move into a python logger, but will remove print statement for now

File: onthequad/util/user_helper.py (hunk 7 above)
Comment (daniellund14) [reply onthequad/util/user_helper.py]: To a better name or different naming convention?

File: onthequad/user_resource.py (hunk 3 above)
Comment (paugajt) [reply onthequad/user_resource.py]: Yeah, we could use some stories for better exception handling

**Pull request merged**